src/serialbox-c/FortranWrapper.cpp

```diff
@@ -25,26 +25,47 @@ using namespace serialboxC;
 using serialbox::Exception;
 using serialbox::TypeID;
 
+namespace {
+void make_4D(std::vector<int>& v) {
+  while(v.size() < 4) {
+    v.push_back(0);
+  }
+}
+
+std::vector<int> make_strides(int istride, int jstride, int kstride, int lstride) {
+  std::vector<int> strides;
+  if(istride >= 0)
+    strides.push_back(istride);
+  if(jstride >= 0)
+    strides.push_back(jstride);
+  if(kstride >= 0)
+    strides.push_back(kstride);
+  if(lstride >= 0)
+    strides.push_back(lstride);
+  return strides;
+}
+}
```

Comment:
Style curiosity: this is closing a namespace, so shouldn't we either

- indent everything inside if the content is short (< 30 lines of code), or
- put a comment to mark the end of a namespace?

Reply:
The indentation is done by clang-format. We can change it, it looks strange to me that the indentation is missing...

 Personally, I don't like conventions if they are not enforced by tools, i.e. the comment in the closing namespace...